src/stars.css

```diff
@@ -0,0 +1,51 @@
+.stars {
+  padding-bottom: 24px;
+
+  display: flex;
+  flex-direction: row;
+
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: var(--product-name-color);
+}
+
+.stars__elem {
+  display: flex;
+  flex-direction: row;
```

Comment:
it's default for display:flex. Apply for other cases as well
```suggestion
```

Reply:
done